navigators/UnAuthNavigator.tsx

```diff
@@ -73,6 +75,11 @@ export default function UnAuthNavigator() {
         component={ResetPassword}
         options={{headerTitle: 'Reset Password'}}
       />
+      <Stack.Screen
+        name="Test"
+        component={TestScreen}
+        options={{headerTitle: 'Reset Password'}}
+      />
```

Comment:
I persume it's a dev screen for testing indeed 😄

Reply:
Si